library/src/main/java/com/okta/oidc/OktaAuthenticationActivity.java

```diff
@@ -289,6 +289,10 @@ private void sendResult(int rc, Intent intent) {
             mResultSent = true;
             setResult(rc, intent);
             finish();
+            if (mCustomTabOptions != null && mCustomTabOptions.getEndEnterResId() != 0
```

Comment:
shoudl this be before finish()?

Reply:
from Java doc:
Call immediately after one of the flavors of {@link #startActivity(Intent)} or {@link #finish} to specify an explicit transition animation to perform next.